src/main/resources/db/changelog/changes/v1.1.0/insert-organizations-contact-info-entity-type.xml

```diff
@@ -0,0 +1,296 @@
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+  <changeSet id="[MODFQMMGR-176] Create entity type for organizations contact info" author="novercash@ebsco.com" runOnChange="true">
+    <!-- Insert the row, then fill it in later, for readability. -->
+    <sql>
+      INSERT INTO entity_type_definition (id, derived_table_name, definition)
+      VALUES ('7a7860cd-e939-504f-b51f-ed3e1e6b12b9', 'drv_organization_contacts', '{}') ON CONFLICT (id) DO NOTHING;
+    </sql>
+
+    <update tableName="entity_type_definition">
+      <column name="definition">
+        {
+          "id": "7a7860cd-e939-504f-b51f-ed3e1e6b12b9",
+          "name": "drv_organization_contacts",
+          "private": false,
+          "fromClause": "src_organizations AS org",
+          "columns": [
+            {
+              "name": "id",
+              "dataType": {
+                "dataType": "rangedUUIDType"
+              },
```

Comment:
you need to ass` "isIDColumn":true`. Then only you will be able to retrieve the results